$MirOS: ports/net/djbdns/patches/patch-src_pickdns-conf_c,v 1.1.201.1 2005/05/14 23:21:04 tg Exp $
--- src/pickdns-conf.c.orig	Sun Feb 11 21:11:23 2001
+++ src/pickdns-conf.c	Sat May 14 22:27:18 2005
@@ -47,7 +47,11 @@ int main(int argc,char **argv)
   start("run");
   outs("#!/bin/sh\nexec 2>&1\nexec envuidgid "); outs(user);
   outs(" envdir ./env softlimit -d250000 ");
+#ifndef DJB_V6ONLY
   outs(auto_home); outs("/bin/pickdns\n");
+#else
+  outs(auto_home); outs("/bin/pickdnst6\n");
+#endif /* DJB_V6ONLY */
   finish();
   perm(0755);
 
